13/umbraco-cms/reference/configuration/connectionstringssettings.md

```diff
@@ -33,5 +33,5 @@ If you're using Umbraco 9 [SQL Server Compact database](https://www.connectionst
 {% endhint %}
 
 ## Provider name
-Because Umbraco cannot determine the provider name from the connection string in all cases. Umbraco follows [Microsoft's convention](ttps://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-5.0#connection-string-prefixes-1) for provider names, which involves specifying it as a postfix in the connection string name.
+Because Umbraco cannot determine the provider name from the connection string in all cases. Umbraco follows [Microsoft's convention](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-5.0#connection-string-prefixes-1) for provider names, which involves specifying it as a postfix in the connection string name.
```

Comment:
Umbraco 13 uses .net8 so I just changed the URL to .net instead of .net 5 😊 
```suggestion
Because Umbraco cannot determine the provider name from the connection string in all cases. Umbraco follows [Microsoft's convention](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-8.0#connection-string-prefixes-1) for provider names, which involves specifying it as a postfix in the connection string name.
```

Reply:
great catch I totally missed that 😵‍💫 thanks for merging in 😄

Reply:
no worries 😉 I have updated as well the links for v10 and v12 with the correct .NET version 👍 

Thank you again for this PR #h5YR